app/controllers/show-geography.js

```diff
@@ -87,15 +88,16 @@ export default class ShowGeographyController extends ParachuteController {
 
   @action
   mutateArray(key, value) {
-    const values = this.get(key);
-
+    let values = this.get(key);
```

Comment:
Confused by this change - is this because values is considered "mutable" through `push` and `remove`?

Reply:
Fixed. (Was leftovers from unnecessary attempt at changing that function.)